articles/appliance/index.html

```diff
@@ -0,0 +1,137 @@
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/geo-ha">Overview of the Geographic High-Availability Appliance</a>
+      </li>
+    </ul>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/modules">Appliance Deployment Modules</a>
+    <p>
+      For security reasons, rules and custom database connections for Auth0 appliances run in a JavaScript sandbox. You can use the full power of the ECMAScript 5 language and a few selected libraries.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/infrastructure">Appliance Infrastructure Requirements</a>
+    <p>
+      This document contains information about the Appliance and its infrastructure requirements.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/appliance-sprint">Appliance Sprint (Customer Onboarding)</a>
+    <p>
+      Appliance Sprint is Auth0’s onboarding program for enterprise customers choosing an Auth0 Appliance. It helps you achieve value quickly with your Auth0 enterprise subscription. The 6 steps in the onboarding program set both you and Auth0 up for success.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/private-cloud-requirements">Information Requirements for Setting Up the Appliance in Auth0's Private Cloud</a>
+    <p>
+      If you have decided to purchase an Appliance that is hosted in a dedicated area of Auth0's cloud, Auth0 will set up the Appliance on your behalf. To do so, Auth0 asks that you provide the following information:
+    </p>
+  </li>
+</ul>
+
+<h2>Appliance Administration</h2>
+
+<!--
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/"></a>
+    <p>
+      
+    </p>
+  </li>
+-->
+
+<ul class="topic-links">
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="https://auth0.com/changelog/appliance">Appliance Release Notes</a>
+    <p>
+      This document shows the latest updates to the Auth0 Appliance.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/admin">Appliance Administrator's Manual</a>
+    <p>
+      This document covers factors Auth0 Appliance administrators should be aware of when working with production Appliance instances.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/dashboard">Appliance Dashboard/Configuration Area</a>
+    <p>
+      Regardless of whether you choose to use Auth0's cloud services or to run Auth0 through Appliance instances, the Management Dashboard is the interface through which you may configure things like your Applications, Connections, Users, and Rules.
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/custom-domains">Custom Domains</a>
+      </li>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/clock">Time Synchronization in the Appliance</a>
```

Comment:
I believe that is a known issue (or intentional) but @ntotten would have to say for sure.